FreeRADIUS/freeradius/_meta/smart-descriptions.json

```diff
@@ -0,0 +1,111 @@
+[
+  {
+    "value": "The user {user.name} logged in from {network.name} with ip {source.ip} and port {source.port}",
+    "conditions": [
+      {
+        "field": "user.name"
+      },
+      {
+        "field": "source.ip"
+      },
+      {
+          "field": "source.port"
+      },
+      {
+          "field": "network.name"
+      },
+      {
+        "field": "freeradius.outcome",
+        "value": "Login OK"
+      }
+    ]
+  },
+  {
+    "value": "The user {user.name} logged in from {network.name} with mac {source.mac} and port {source.port}",
```

Comment:
Just out of curiosity, how is it possible to have a `source.port` and a missing `source.ip` ? :thinking:

Reply:
Yes. For application authentication, the logs only supply mac address and port (see [this example](https://github.com/SEKOIA-IO/intake-formats/blob/feat/FreeRadius/FreeRADIUS/freeradius/tests/test_login_ok1.json#L9))